Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1092 @@
+        return response
+
+    def retrieve_closed_incidents_request(self, page, items, adversary_types, labels):
+        params = assign_params(page=page, items=items, key=self.api_key)
+        data = {"adversary-types": adversary_types, "labels": labels}
+        headers = self._headers
+
+        response = self._http_request(
+            "POST",
+            "api/incidents/closed",
+            params=params,
+            json_data=data,
+            headers=headers,
+        )
+
+        return response
+
+    def retrieve_endpoints_by_incident_request(self, lumu_incident_id, page, items):
+        params = assign_params(page=page, items=items, key=self.api_key)
+        headers = self._headers
+
+        response = self._http_request(
+            "POST",
+            f"api/incidents/{lumu_incident_id}/endpoints-contacts",
+            params=params,
+            headers=headers,
+        )
+
+        return response
+
+    def mark_incident_as_read_request(self, lumu_incident_id):
+        params = assign_params(key=self.api_key)
+        headers = self._headers
+
+        response = self._http_request(
+            "POST",
+            f"api/incidents/{lumu_incident_id}/mark-as-read",
+            params=params,
+            headers=headers,
+        )
+
+        return response
+
+    def mute_incident_request(self, lumu_incident_id, comment):
+        params = assign_params(key=self.api_key)
+        data = {"comment": comment}
+        headers = self._headers
+
+        response = self._http_request(
+            "POST",
+            f"api/incidents/{lumu_incident_id}/mute",
+            params=params,
+            json_data=data,
+            headers=headers,
+        )
+
+        return response
+
+    def unmute_incident_request(self, lumu_incident_id, comment):
+        params = assign_params(key=self.api_key)
+        data = {"comment": comment}
+        headers = self._headers
+
+        response = self._http_request(
+            "POST",
+            f"api/incidents/{lumu_incident_id}/unmute",
+            params=params,
+            json_data=data,
+            headers=headers,
+        )
+
+        return response
+
+    def consult_incidents_updates_through_rest_request(self, offset, items, time):
+        params = assign_params(offset=offset, items=items, time=time, key=self.api_key)
+        headers = self._headers
+
+        response = self._http_request(
+            "GET",
+            "api/incidents/open-incidents/updates",
+            params=params,
+            headers=headers,
+        )
+
+        return response
+
+    def close_incident_request(self, lumu_incident_id, comment):
+        params = assign_params(key=self.api_key)
+        data = {"comment": comment}
+        headers = self._headers
+
+        response = self._http_request(
+            "POST",
+            f"api/incidents/{lumu_incident_id}/close",
+            params=params,
+            json_data=data,
+            headers=headers,
+        )
```

Comment:
no need to pass the `self._headers` it use it by default
please change in all the places 
```suggestion

        response = self._http_request(
            "POST",
            f"api/incidents/{lumu_incident_id}/close",
            params=params,
            json_data=data,
        )
```

Reply:
done in this f47ed2c15d6f2c181b37fe28e7a84706396c6845